lib/modules/side_menu/components/user.dart

```diff
@@ -0,0 +1,48 @@
+part of '../side_menu_module.dart';
+
+class User extends StatelessWidget {
+  const User({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final state = context.findAncestorStateOfType<_SideMenuViewImplState>()!;
+
+    return StreamsSelector0<UserInfo>.value(
+        stream: state._user,
+        builder: (_, user, __) {
+          return Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Flexible(
+                child: Text(
+                  user.email!,
+                  maxLines: 10,
+                  style: const TextStyle(
+                    color: Colors.white,
+                    fontSize: 34,
+                    fontWeight: FontWeight.w800,
```

Comment:
maybe you will want to manage these style from the Theme level later (theme refactor)

Reply:
@sleepylee putted in my list 😓